src/components/Widget/FileWidget.jsx

```diff
@@ -119,6 +119,23 @@ const FileWidget = (props) => {
       }
     };
     reader.readAsDataURL(files[0]);
+
+    /* add dummy file to input element,
+       to make sure the browsers form validation
+       recognizes this field as filled
+    */
+    let inputEl = document.getElementById(`field-${id}`);
+    const myFile = new File([], file.name, {
+      type: 'text/plain',
```

Comment:
as i understand, you are forcing the file mime_type. What appens if i upload a jpg, or png, or pdf or xls file for example? could you get mime type from file var?